tracer/src/Datadog.Trace/Ci/Agent/MessagePack/CIEventMessagePackFormatter.cs

```diff
@@ -30,16 +30,7 @@ internal class CIEventMessagePackFormatter : EventMessagePackFormatter<EventsPay
 
         public CIEventMessagePackFormatter(TracerSettings tracerSettings)
         {
-            var environment = tracerSettings.Environment;
-            if (environment is not null)
-            {
-                _environmentValueBytes = StringEncoding.UTF8.GetBytes(environment);
-            }
-            else
-            {
-                _environmentValueBytes = null;
-            }
-
+            _environmentValueBytes = StringEncoding.UTF8.GetBytes(tracerSettings.Environment ?? "none");
```

Comment:
Doesn't "none" in `env` have a special meaning 🤔? Are we sure this is safe?

Reply:
yes, is the default when is not set: https://github.com/DataDog/datadog-agent/blob/cbac965c53670c0cee8f1d6f736848093f375035/pkg/trace/testutil/stats.go#L14